narayana/scripts/hudson/bm.sh

```diff
@@ -23,12 +23,12 @@ function cmp_narayana {
 
 function runs {
 
-  if [ -z ${THREAD_COUNTS+x} ]
+  if [ -z "${THREAD_COUNTS}" ]
   then
     THREAD_COUNTS="1 24 240 1600"
   fi
 
-  if [ -z ${BF_COUNTS+x} ]
+  if [ -z "${BF_COUNTS}" ]
```

Comment:
I found that it seemed like if BF_COUNTS was present in the shell (but empty) this check was false and so no BF_COUNTS were set. I decided to make the same change to check the variable in quotes to THREAD_COUNTS

Reply:
ok